src/services/contactReport.service.js

```diff
@@ -0,0 +1,29 @@
+import { $db } from '@/services/firebase.service'
+// import { roomService } from '@/services/room.service'
+class ContactReportService {
+  async getContactsOfUserByID (userId, date) {
+    // let myRoom
+    const userList = []
+    const userRef = await $db().doc('User/' + userId)
+    const querySnapshot = await $db().collectionGroup('CheckIn')
+      .where('date', '==', date).get()
```

Comment:
Maybe you can check here if the array contains the user

```suggestion
      .where('date', '==', date)
      .where('user', 'array-contains', userRef).get()
```

Reply:
that's it! THX